ykrt/src/compile/jitc_yk/codegen/x64/mod.rs

```diff
@@ -437,12 +437,15 @@ impl<'a> Assemble<'a> {
                     .get_ready_for_deopt(&mut self.asm, &self.deoptinfo[&deoptid].0);
                 // FIXME: Why are `deoptid`s 64 bit? We're not going to have that many guards!
 
-                // Word-align this location so we can patch this later in a single atomic
-                // operation.
+                // Align this location to the cache-line, which since a cache-line is a multiple of
+                // 8 bytes, will also word-align it too. This allows us to safely patch this place
+                //   later in a single atomic operation.
                 let off = self.asm.offset().0;
+                let cachesize =
```

Comment:
I would prefer to call this `clsize` or something that doesn't suggest it's the *cache's* size. [I know, I know, bikeshedding, but this is easily misunderstood.]

Reply:
[9046424](https://github.com/ykjit/yk/pull/1591/commits/9046424d13ed52469418f28e9ce8b3d2d764c04d)